routes/ssrf.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const {
+  sinks: { ssrf }
+} = require('@contrast/test-bench-utils');
+
+const EXAMPLE_URL = 'http://www.example.com';
+
+module.exports = ({ router }) => {
+  router.get('/ssrf', (ctx) =>
+    ctx.render('../node_modules/@contrast/test-bench-content/views/ssrf', {
```

Comment:
this is unfortunate. there has to be a way to get that string from requiring of the test-bench-content module

Reply:
looks like maybe we can use `renderFile`?

Reply:
oh that's not even an npm module

Reply:
This was meant as a temporary solution until _all_ shared templates lived in the new repo and we could then configure the view engine to use the package folder as the source of views. I agree that this isn't great. I can continue and see if there's a better short-term solution.

Reply:
This was resolved. A rule-specific template was installed that pulled in the view from the template. This way is at least an improvement I think.